web/app/components/since.js

```diff
@@ -1,26 +1,26 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect } from "react";
 
-export default function Since( { props }) {
-  const [time, setTime] = useState(new Date());
-  const since = new Date(props);
+export default function Since({ lastEaten }) {
+  const [now, setNow] = useState(new Date());
 
   useEffect(() => {
     const timer = setInterval(() => {
-      setTime(new Date());
-    }, 1000)
+      setNow(new Date());
+    }, 1000);
     return () => {
-      clearInterval(timer)
-    }
-  }, [])
-  
-  // const daysSince = Math.floor((time - since) / (1000 * 24 * 60 * 60));
-  const secondsSince = Math.floor((time - since) / (1000));
-  
-  return ( 
+      clearInterval(timer);
+    };
+  }, []);
+
+  const secondsSince = lastEaten
```

Comment:
the variable name `secondsSince` communicates to me that it holds a numerical value representing time in seconds since eaten. If we're using this to store the display value which might be also be text it would be best IMO to name it something that keeps that possibility open in the mind of a developer reading this. So maybe something like `timeSinceEatenMessage`?

A second concern I have with this is that we might want to display an entirely different node/component here if the allergen was never eaten. So maybe `secondsSinceEaten` makes sense (and we might want to call this `timeSinceEaten` to be more generic so that when we switch to days we don't have to rename variables), but we use a conditional based on the existence of `lastEaten` to swap out the entire contents of what's displayed here. That way we either show "Seconds since last eaten: [TIME]" or "Never eaten"

Reply:
I've changed it to timeSinceEaten, and made the message dependent on the truthiness of lastEaten, since that does make more sense.